copc/lidar-las-to-copc.ipynb

```diff
@@ -0,0 +1,707 @@
+{
```

Comment:
<p>Wording suggestion:</p><blockquote>To access NASA EarthData into Jupyter Notebook, you can create an account by visiting <a href="https://urs.earthdata.nasa.gov/users/new" target="_blank">NASA's Earthdata Login page</a>. This will enable you to register for an account and retrieve the datasets used in the notebook.</blockquote><p>To access NASA EarthData in Jupyter you need to register for an [Earthdata account](<a href="https://urs.earthdata.nasa.gov/users/new" target="_blank">https://urs.earthdata.nasa.gov/users/new</a>).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cloudnativegeo/cloud-optimized-geospatial-formats-guide/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Noted with thanks. I've updated it in the revision.</p>